LayoutTestBase/Config/LYTConfig.h

```diff
@@ -103,6 +103,8 @@ NS_ASSUME_NONNULL_BEGIN
 
  Defaults:
  - UISwitch
+ - UIButton
```

Comment:
Total nitpick. I would switch the order of UIButton and UITextView so it matches the documentation comments. But no big deal. Otherwise ship it!